paper-toggle-button.html

```diff
@@ -134,12 +93,29 @@
         transition: none;
       }
 
+      :host([checked]):not([disabled]) .toggle-bar {
+        opacity: 0.5;
+        background-color: var(--paper-toggle-button-checked-bar-color, --default-primary-color);
+        @apply(--paper-toggle-button-checked-bar);
+      }
+
+      :host([disabled]) .toggle-bar {
+        background-color: #000;
+        opacity: 0.12;
+      }
+
       :host([checked]) .toggle-button {
         -webkit-transform: translate(16px, 0);
         transform: translate(16px, 0);
       }
 
+      :host([checked]):not([disabled]) .toggle-button {
+        background-color: var(--paper-toggle-button-checked-button-color, --default-primary-color);
+        @apply(--paper-toggle-button-checked-button);
+      }
+
       :host([disabled]) .toggle-button {
+        /* Need !important to override the value set in `:host([checked]) .toggle-button` */
```

Comment:
Also, it should would work fine to do something along the lines of:

```css
:host([disabled]) .toggle-button,
:host([disabled][checked]) .toggle-button {
  /* ... */
}
```

If you want to be explicit. Otherwise, maybe make a note about how the ordering is important?